### PR TITLE
AP_Airspeed: Add additional DLVR support 20inH2O 30inH2O 60inH2O

### PR DESCRIPTION
**Pull request by JoshWelsh**
Hitec is developing the 20 and 60 versions and will likely do a 30 in the future.

edit: backport requested

### CODE REVIEW

**Comment by IamPete1** [comment]
The type param description needs updating too

**Comment by JoshWelsh** [comment]
> The type param description needs updating too

Ah thanks @IamPete1 I had missed that, but I think I've got it pushed back up properly.

File: libraries/AP_Airspeed/AP_Airspeed.cpp [hunk 1]
```diff
@@ -70,7 +70,7 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: Airspeed type
     // @Description: Type of airspeed sensor
-    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X,7:I2C-DLVR-5in,8:UAVCAN,9:I2C-DLVR-10in
+    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X,7:I2C-DLVR-5in,8:UAVCAN,9:I2C-DLVR-10in,10:I2C-DLVR-20in,10:I2C-DLVR-30in,10:I2C-DLVR-60in
```
Comment (IamPete1) [inline libraries/AP_Airspeed/AP_Airspeed.cpp:73]:
```suggestion
    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X,7:I2C-DLVR-5in,8:UAVCAN,9:I2C-DLVR-10in,10:I2C-DLVR-20in,11:I2C-DLVR-30in,12:I2C-DLVR-60in
```

Comment (IamPete1) [reply libraries/AP_Airspeed/AP_Airspeed.cpp:73]: Also _TYPE2

**Comment by JoshWelsh** [comment]
I just squished all the edits, hopefully I did that right as well.  Thanks @IamPete1 for finding the miss on the type params!

**Pull request merged**